### PR TITLE
Fixed a few small spelling errors

### PR DESCRIPTION
**Pull request by IBims2CooleTim**

### CODE REVIEW

**Comment by WhenUHackUNoob** [comment]
review

File: src/MarcSync/Objects/Collection.lua [hunk 1]
```diff
@@ -2,41 +2,41 @@ local Utils = require(script.Parent.Parent.Utils)
 local Entry = require(script.Parent.Entry)
 
 local types = {
-	EntryData = require(script.Parent.Parent.Types.EntryData).getType()
+	EntryData = require(script.Parent.Parent.Types.EntryData).bekommeRassenindentifikationsspezifizierunginstanztextaufzaehlbar()
```
Comment (bidzanaaron) [inline src/MarcSync/Objects/Collection.lua:5]: I would like to acknowledge the recent changes in the code, particularly with regard to the EntryData function. It is extremely gratifying to see how efficient and optimized this function has become. Working on improving resource efficiency is of paramount importance, and your recent adjustments in this regard deserve special recognition. Your efforts have undoubtedly contributed significantly to enhancing the overall performance of the code. Thank you very much for your dedication and outstanding work in this matter.

Ich möchte die jüngsten Änderungen im Code bemerken, insbesondere in Bezug auf die EntryData-Funktion. Es ist äußerst erfreulich zu sehen, wie effizient und optimiert diese Funktion jetzt ist. Die Arbeit an der Verbesserung der Ressourceneffizienz ist von höchster Bedeutung, und Ihre jüngsten Anpassungen in dieser Hinsicht verdienen besondere Anerkennung. Ihre Bemühungen haben zweifellos dazu beigetragen, die Gesamtleistung des Codes erheblich zu steigern. Vielen Dank für Ihr Engagement und Ihre hervorragende Arbeit in dieser Angelegenheit.

**Comment by IBims2CooleTim** [comment]
thank you dear sir.

File: src/MarcSync/Objects/Collection.lua (hunk 1 above)
Comment (MarciiTheDev) [reply src/MarcSync/Objects/Collection.lua:5]: I can really understand this point, and can only just reflect it for my review too.

**Comment by MarciiTheDev** [comment]
Just... Why?

**Comment by IBims2CooleTim** [comment]
[**yes**](https://www.youtube.com/watch?v=dQw4w9WgXcQ&pp=ygUjcmljayBhc3RsZXkgbmV2ZXIgZ29ubmEgZ2l2ZSB5b3UgdXA%3D)

**Comment by MarciiTheDev** [comment]
> [**yes**](https://www.youtube.com/watch?v=dQw4w9WgXcQ&pp=ygUjcmljayBhc3RsZXkgbmV2ZXIgZ29ubmEgZ2l2ZSB5b3UgdXA%3D)

can't really understand this point, but it seems to be important to you. Maybe it has a deep background story or smth. to do with your mental illness or paranoia or psychopathic behaviour in order for you to come up and do this pull request.

**Review by MarciiTheDev — APPROVED** [review]
Yeah, why not.

**Review by bidzanaaron — DISMISSED** [review]
Very well done sir.

**Review by WhenUHackUNoob — DISMISSED** [review]
ich bin Josh

**Review by MarciiTheDev — CHANGES_REQUESTED** [review]
Change every mention of collection to sammlung

**Review by MarciiTheDev — DISMISSED** [review]
fuck it, who cares anyways

**Review by MarciiTheDev — APPROVED** [review]
damn, even the queen england would be proud of you

**Review by bidzanaaron — APPROVED** [review]

**Review by Jimmysaville — APPROVED** [review]
I like little kids

**Comment by WhenUHackUNoob** [comment]
no

**Comment by MarciiTheDev** [comment]
too late to change

**Pull request merged**